test/indexer/test_hca_indexer.py

```diff
@@ -773,11 +773,13 @@ def test_metadata_generator(self):
     def test_metadata_field_exclusion(self):
         self._index_canned_bundle(self.old_bundle)
 
-        # Check that the dynamic mapping has the field disabled
+        # Check that the dynamic mapping for the field is present
         bundles_index = config.es_index_name('bundles')
         mapping = self.es_client.indices.get_mapping(index=bundles_index)
-        self.assertFalse(
-            mapping[bundles_index]['mappings']['doc']['properties']['contents']['properties']['metadata']['enabled'])
+        self.assertIn('bundle_uuid', mapping[bundles_index]
+                        ['mappings']['doc']['properties']
+                        ['contents']['properties']
+                        ['metadata']['properties'])
```

Comment:
Wrapping of the nested keys should align under the 'm' of `mapping` or wrap the parameters of `assertIn` to avoid the need to split the nested keys on multiple lines
```suggestion
        self.assertIn(
            'bundle_uuid',
            mapping[bundles_index]['mappings']['doc']['properties']['contents']['properties']['metadata']['properties']
        )
```